Minor fixes (#262)
SlimeVR/SlimeVR-Docs-Site: src/quick-setup.md

   ![The first page of the SlimeVR Wizard](assets/img/quick_intro.png)

> 1. Input your 2.4GHz Wi-Fi credentials, so that your tracker can connect to WiFi, then click **Submit**.

Reviewer: ```suggestion
1. Input your 2.4GHz Wi-Fi credentials, so that your tracker can connect to Wi-Fi, then click **Submit**.
```

   ![Inputting WiFi credentials](assets/img/quick_wifi.png)

1. Plug your trackers in one at a time and turn them on, you should see the progress bar on the left update to show the Wi-Fi details being sent. Make sure you use the cable included with your trackers, as other cables may not be suited to send data.

	![Animation of the connection process](assets/img/quick_connectTracker.gif)

> 1. Once you have connected all your trackers, you should see them listed with numbers on the right. If you have forgotten which trackers have yet to be plugged in, shaking a powered on tracker will highlight it in the list, if it has already been connected. Click **I connected all my trackers** once you are finished.

Reviewer: ```suggestion
1. Once you have connected all your trackers, you should see them listed with numbers on the right. If you have forgotten which trackers have yet to be plugged in, shaking a powered on tracker will highlight it in the list if it has already been connected. Click **I connected all my trackers** once you are finished.
```

Author: Yeah I couldn't find a way nice to word it when I was working on this, this is more clear though

1. Figure out which body parts you will need to assign. Depending on how many trackers you have these are the suggested locations:

	* Lower-Body Set (5 trackers) - Chest, both thighs, both ankles.
	* Core Set (5 trackers with one extension) - Chest and hip/waist (via the tracker with an extension), both thighs, both ankles.
	* Enhanced Core Set (5 trackers with three extensions) - Chest and hip/waist (via trackers with extension), both thighs, both ankles and feet (via trackers with extensions).

> 	* Full-Body Set (7 trackers with three extensions) - Both upper arms, chest and hip/waist (via trackers with extension), both thighs, both ankles and feet (via trackers with extensions).

Reviewer: This isn't necessarily wrong, but should we recommend waist? I've found hip performs better in general, but this is kinda a personal opinion thing when it comes to this.

Author: I do agree that hip performs better, but there were people in the Discord who struggled to get good mounting on hip and weren't aware that you can use waist instead

Reviewer: Understandable, I just think people might be attracted towards using "waist" instead as they know "waist tracking" from SteamVR stuff. Either way this is fine, I'll just leave this up for consideration but this isn't really a big issue.

	![Reset Tutorials](assets/img/quick_reset.png)

	* **Tapping Chest** - Yaw reset, resets the trackers to assume they are facing their defined mounting direction.
	* **Tapping Left Thigh** -	Full reset, resets the trackers to the assumption that you are in an I-pose.

> 	* **Tapping Right Thigh** - Mounting reset, resets the trackers mounting. You must be in the ski position you took for the mounting wizard for this to work.  

Reviewer: Weird wording, assumes the user has done something they may not have done
```suggestion
	* **Tapping Right Thigh** - Mounting reset, resets the trackers to an estimated mounting direction. You must be in the ski position as illustrated in the mounting calibration wizard for this to work.
```
